lib/sqlalchemy/dialects/postgresql/psycopg2.py

```diff
@@ -52,11 +52,42 @@
 
     :ref:`psycopg2_unicode`
 
-* ``use_batch_mode``: This flag allows ``psycopg2.extras.execute_batch``
-  for ``cursor.executemany()`` calls performed by the :class:`.Engine`.
-  It is currently experimental but
-  may well become True by default as it is critical for executemany
-  performance.
+.. _psycopg2_executemany_mode:
```

Comment:
**mike bayer** (zzzeek) wrote:

I'm reworking the organization here since this is inside of a bulleted list stating all parameters,